docs/claims.md

```diff
@@ -16,30 +16,36 @@ The HATVaults contract only implements some time-out checks in case the arbitrat
 
 ## Details and restrictions
 
+There are 3 "periods" relevant for the handling of a new claim:
+1. Challenge Period: the first `challengePeriod` seconds after the submission of the claim 
+1. Challenge Timeout Period: the `challengeTimeOutPeriod` seconds after the challenge period ended
+1. Expiration: after `challengePeriod + challengeTimeoutPeriod` seconds after the submission of the claim
+
 1. **SUBMISSION** 
   `submitClaim(beneficiary, bountyPercentage)` will create a new `claimId`.
    - `submitClaim` can only be called by the committee
    - `submitClaim` can only be called during a safety period
    - `submitClaim` can only be called if no other active claim exists
    - emergencyPause must not be in effect
 1. **CHALLENGE** `challengeClaim(_claimId)` 
-   - can only be called during the challengePeriod
+   - can only be called during the challenge period
    - only the `arbitrator` or the `registry.owner` can challenge a claim (the registry.owner functions here as a "challenger of last resort")
    - can only be called if the claim has not been challenged yet
-   - `_claimId` must be the id of the currently active claim
```

Comment:
too obvious?

Reply:
read it with a stresss on "currently active"